styles/main.css

```diff
@@ -130,6 +129,8 @@ footer {
     border-bottom: 1px solid rgba(0,0,0,0.3);
     box-shadow: 0 0 4px rgba(255,255,255,0.2); 
     background-color: pink;
+    display: flex;
+    flex-direction: column;

```

Comment:
You need these here in order to get the footer's auto margin to get pushed to the bottom